inc/graph.class.php

```diff
@@ -277,4 +277,147 @@ private function checkEmptyLabels(&$labels) {
          }
       }
    }
+
+   public function showAgentOnlineStats() {
+      global $DB;
+
+      $domain = $range = $serie = [];
+      $FlyvemdmAgent = PluginFlyvemdmAgent::getTable();
+      $logTable = Log::getTable();
+      $query = "SELECT l.`new_value` AS online_date, f.`name` AS agent_name, f.`id` AS agent_id 
```

Comment:
If you introduce queries un the plugin, use the query builder. RAW SQL queries is deprecated.

http://glpi-developer-documentation.readthedocs.io/en/master/devapi/database/index.html

Reply:
done it on new commit